src/components/SubscribeNewsletter/subscribe-newsletter.tsx

```diff
@@ -0,0 +1,40 @@
+import { SquiggleLeft, SquiggleRight } from '@/assets/svg'
+import Button from '../Button/button'
+
+function SubscribeNewsLetter() {
+  return (
+    <div className="w-full relative bg-ibtc-light-yellow flex flex-col items-center justify-center min-h-[26rem] gap-[2rem]">
```

Comment:
The gap between the children components should be 4rem = 64px/16px and not 2rem
![Screenshot (45)-64px](https://user-images.githubusercontent.com/51183064/231300585-326dfaa6-057e-434a-9a74-a6c89ea44e99.png)